client/my-sites/domains/domain-management/list/domain-row.jsx

```diff
@@ -152,21 +152,27 @@ class DomainRow extends PureComponent {
 	}
 
 	renderMobileExtraInfo( expiryDate, domainTypeText ) {
-		const { domain } = this.props;
+		const { domain, translate } = this.props;
 
 		let extraInfo = '';
 		if ( domainTypeText ) {
 			extraInfo = domainTypeText;
 		} else if ( domain.expired ) {
 			if ( expiryDate ) {
-				extraInfo = 'Expired on ' + expiryDate.format( 'LL' );
+				extraInfo = translate( 'Expired on %(expiryDate)s', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Expired: %(expiryDate)s' )` *ES Score: 6*